src/quo2/foundations/customization_colors.cljs

```diff
@@ -0,0 +1,19 @@
+(ns quo2.foundations.customization-colors
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]))
+
+(defn get-overlay-color
+  [theme pressed?]
+  (if (= theme :light)
+    (colors/alpha colors/black (if-not pressed? 0 0.2))
```

Comment:
A bit simpler `(if pressed? 0.2 0)`

Reply:
I agree, but it's just when I think of a button I would put the non pressed value first. That's why it's like that in the rest of this pr too.

Reply:
happy to change ofc for whatever makes most sense. 👍

Reply:
It's not a big deal @J-Son89, just slightly more mental effort to take the negation in consideration.

For instance, sometimes, the reader won't read from left to right, i.e. the reader will look at the second argument (`0.2`) and ask, when is this value returned? In this case, the reader has to negate the negation, i.e. *0.2 is returned when "if-not pressed?" is false, which is simply when "pressed?" is true*. This is a contrived example of course, but in other cases avoiding the negation in the condition leads to code that's easier to understand.